opencti-platform/opencti-graphql/src/migrations/1703150526503-user-confidence-level.js

```diff
@@ -0,0 +1,54 @@
+import { elUpdateByQueryForMigration, elUpdateIndicesMappings } from '../database/engine';
+import { READ_INDEX_INTERNAL_OBJECTS } from '../database/utils';
+import { DatabaseError } from '../config/errors';
+
+export const up = async (next) => {
+  await elUpdateIndicesMappings({
+    user_confidence_level: {
+      properties: {
+        max_confidence: { type: 'integer' },
+        overrides: {
+          type: 'nested',
+          properties: {
+            entity_type: { type: 'keyword' },
```

Comment:
type "keyword" is the right type I think, but you should probably check with Julien what he is planning when enforcing the mapping with these properties. Usually with string fields it's mapped like this: 

```
type: 'text',
fields: {
  keyword: {
    type: 'keyword',
    normalizer: 'string_normalizer',
    ignore_above: 512,
  },
}
```

Reply:
I'm not sure, I'll check with him indeed.

our `string_normalizer` is doing a lowercase for instance ; right for the ids but not for the entity type which are upper cased like 'Report'.